src/test/resources/application-integration.yml

```diff
@@ -1,3 +1,13 @@
+spring:
+  datasource:
+    driver-class-name: org.postgresql.Driver
+    url: jdbc:postgresql://localhost:5432/${POSTGRES_DB}
```

Comment:
```suggestion
    url: jdbc:postgresql://db_postgres:5432/${POSTGRES_DB}
```
or
```suggestion
    url: jdbc:postgresql://${DB_HOST:db_postgres}:5432/${POSTGRES_DB}
```
and proper variable values

Docker: `DB_HOST=db_postgres`
Localhost: `DB_HOST=localhost`